pkgs/data/fonts/apple-color-emoji/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, lib, fetchFromGitHub, python3, zopfli, pngquant, imagemagick, which }:
+
+let
+  python = python3.withPackages (
+    ps: with ps; [
+      fonttools
+      nototools
+    ]
+  );
+
+in
+stdenv.mkDerivation rec {
+  pname = "apple-color-emoji";
+  version = "unstable-2020-04-13";
+
+  src = fetchFromGitHub {
+    owner = "samuelngs";
+    repo = "apple-emoji-linux";
+    rev = "8c247a77eb0abe9fca1011a6ab75520dc761ff37";
+    sha256 = "1r0laf5g4gjrrv2xzr1l5bvkn7px9k02p6wknxaafm3mrmpxrk23";
+  };
+
+  dontConfigure = true;
+
+  nativeBuildInputs = [ python zopfli pngquant imagemagick which ];
+
+  enableParallelBuilding = true;
+
+  patchPhase = ''
+    substituteInPlace Makefile --replace 'HOME/.local/' 'out/'
```

Comment:
With [this patch](https://github.com/samuelngs/apple-emoji-linux/pull/7), you can just use `makeFlags = [ "PREFIX=${placeholder "out"}" ];`

Reply:
Thanks! So, should I wait for PR merge and make changes after that?

Reply:
PR is merged :)

Reply:
Thanks for reminding me! bad news, I changed to latest rev, tried to build. still it failes to build with this error.
Same error as [GitHub CI](https://github.com/samuelngs/apple-emoji-linux/runs/1036473672?check_suite_focus=true)
```
Dropped outline ('glyf', 'CFF ') and related tables.
Output font 'AppleColorEmoji.ttf' generated.
Traceback (most recent call last):
  File "map_pua_emoji.py", line 73, in <module>
    main(sys.argv)
  File "map_pua_emoji.py", line 69, in main
    add_pua_cmap(argv[1], argv[2])
  File "map_pua_emoji.py", line 61, in add_pua_cmap
    glyph_name = get_glyph_name_from_gsub([ch1, ch2], font)
  File "map_pua_emoji.py", line 43, in get_glyph_name_from_gsub
    ligatures = lookup.SubTable[0].ligatures
  File "/nix/store/pf57mqvrygkxl8r4s75f2fd7qff394ji-python3-3.8.5-env/lib/python3.8/site-packages/fontTools/ttLib/tables/otBase.py", line 548, in __getattr__
    raise AttributeError(attr)
AttributeError: ligatures
make: *** [Makefile:145: AppleColorEmoji.ttf] Error 1
rm AppleColorEmoji.tmpl.ttx build/emoji/emoji_ufe82b.png
builder for '/nix/store/n7mhpi8v4lvpizca88iksi02144ixc8l-apple-color-emoji-unstable-2020-08-27.drv' failed with exit code 2
error: build of '/nix/store/n7mhpi8v4lvpizca88iksi02144ixc8l-apple-color-emoji-unstable-2020-08-27.drv' failed
```
I would like to make a PR to fix if I could, but I'm new to programming in general. I don't know where to start investigating the reason.
Seems like build error starts after [updating emoji icons](https://github.com/samuelngs/apple-emoji-linux/commit/35656af89ce7e1e4b2afc9360eaa32bac2a0d705)

Reply:
I think the issue is that the toolchain relies on outdated versions of fonttools and nototools. The versions we ship in nixpkgs don't even support python 2 anymore which apple-emoji-linux seems to require.

What we could try is to reuse the noto-fonts-emoji toolchain to build apple-color-emoji, similarly to how twitter-color-emoji works in nixpkgs.